android/src/main/java/com/twiliovoicereactnative/TwilioVoiceReactNativeModule.java

```diff
@@ -540,39 +541,53 @@ public void callInvite_reject(String uuid, Promise promise) {
     }
   }
 
-  Call.Score getScoreFromId (int x) {
-    switch(x) {
-      case 0:
-        return Call.Score.NOT_REPORTED;
-      case 1:
-        return Call.Score.ONE;
-      case 2:
-        return Call.Score.TWO;
-      case 3:
-        return Call.Score.THREE;
-      case 4:
-        return Call.Score.FOUR;
-      case 5:
-        return Call.Score.FIVE;
-    }
-    return Call.Score.NOT_REPORTED;
+  /**
+   * Map of common constant score strings to the Call.Score enum.
+   */
+  private final Map<String, Call.Score> scoreMap = Map.of(
+    CommonConstants.CallFeedbackScoreNotReported, Call.Score.NOT_REPORTED,
+    CommonConstants.CallFeedbackScoreOne, Call.Score.ONE,
+    CommonConstants.CallFeedbackScoreTwo, Call.Score.TWO,
+    CommonConstants.CallFeedbackScoreThree, Call.Score.THREE,
+    CommonConstants.CallFeedbackScoreFour, Call.Score.FOUR,
+    CommonConstants.CallFeedbackScoreFive, Call.Score.FIVE
+  );
+
+  /**
+   * Use the score map to get a Call.Score value from a string.
+   * @param score The score as a string passed from the JS layer.
+   * @return a Call.Score enum value. If the passed string is not in the enum, defaults to
+   * Call.Score.NOT_REPORTED.
+   */
+  Call.Score getScoreFromString(String score) {
+    return scoreMap.containsKey(score)
+      ? scoreMap.get(score)
+      : Call.Score.NOT_REPORTED;
   }
 
+  /**
+   * Map of common constant issue strings to the Call.Issue enum.
+   */
+  private final Map<String, Call.Issue> issueMap = Map.of(
+    CommonConstants.CallFeedbackIssueAudioLatency, Call.Issue.AUDIO_LATENCY,
```

Comment:
Move this to where the other private variables are defined and make it static as well (it never changes correct?)